src/sql/workbench/contrib/welcome/notifyHiddenTenant/notifyHiddenTenantDialog.ts

```diff
@@ -0,0 +1,74 @@
+import { ErrorMessageDialog } from 'sql/workbench/services/errorMessage/browser/errorMessageDialog';
+import { Link } from 'vs/platform/opener/browser/link';
+import { IStorageService, StorageScope, StorageTarget } from 'vs/platform/storage/common/storage';
+import { IOpenerService } from 'vs/platform/opener/common/opener';
+import { IConnectionManagementService } from 'sql/platform/connection/common/connectionManagement';
+import { mssqlProviderName } from 'sql/platform/connection/common/constants';
+import { TelemetryView } from 'sql/platform/telemetry/common/telemetryKeys';
+import { NOTIFY_HIDETENANT_SHOWN, NOTIFY_READMORE_LINK } from 'sql/workbench/contrib/welcome/constants';
+import { IAccountManagementService } from 'sql/platform/accounts/common/interfaces';
+
+export class NotifyHiddenTenantDialog extends ErrorMessageDialog {
+
+	constructor(
+		@IThemeService themeService: IThemeService,
+		@IClipboardService clipboardService: IClipboardService,
+		@ILayoutService layoutService: ILayoutService,
+		@IAdsTelemetryService telemetryService: IAdsTelemetryService,
+		@IContextKeyService contextKeyService: IContextKeyService,
+		@ILogService logService: ILogService,
+		@ITextResourcePropertiesService textResourcePropertiesService: ITextResourcePropertiesService,
+		@IOpenerService openerService: IOpenerService,
+		@IInstantiationService private _instantiationService: IInstantiationService,
+		@IStorageService private _storageService: IStorageService,
+		@IConnectionManagementService private _connectionManagementService: IConnectionManagementService,
+		@IAccountManagementService private _accountManagementService: IAccountManagementService
+	) {
+		super(themeService, clipboardService, layoutService, telemetryService, contextKeyService, logService, textResourcePropertiesService, openerService);
+	}
+
+	public override open(): void {
+		if (this._storageService.get(NOTIFY_HIDETENANT_SHOWN, StorageScope.APPLICATION)) {
+			return;
+		}
+
+		this._storageService.store(NOTIFY_HIDETENANT_SHOWN, true, StorageScope.APPLICATION, StorageTarget.MACHINE);
+		this._accountManagementService.getAccounts().then(accounts => {
+			// Do not notify users who don't have any Azure connection in their list of connections and no Azure accounts registered.
+			if (!this._connectionManagementService.getConnections()?.some(conn => conn.providerName === mssqlProviderName
+				&& conn.authenticationType === 'AzureMFA') && (!accounts || accounts.length === 0)) {
+				return;
+			}
+
+			super.open(TelemetryView.NotifyHiddenTenantDialog, Severity.Info,
+				localize('notifyHiddenTenant.title', 'Important Update'),
+				localize('notifyHiddenTenant.message', `Azure Data Studio has made design changes in the way access tokens are fetched from Azure Active Directory. You will no longer need to select an 'Azure AD Tenant' entry when connecting to Azure SQL instances. Tenant information will now be retrieved from the server during login.`, '\n\n'));
```

Comment:
I would rework this text to not directly refer to Azure Data Studio or design changes, as this can be inferred from the context of the message.  Maybe something like the below (or similar) more actively states the same thing?

`Connections using Azure Active Directory authentication will now retrieve tenant information from the server during login.  The 'Azure AD Tenant' entry no longer needs to be provided when connecting to Azure SQL instances.`

Reply:
```suggestion
				localize('notifyHiddenTenant.message', `Connections using Azure Active Directory authentication will now retrieve tenant information from the server during login. The 'Azure AD Tenant' entry no longer needs to be provided when connecting to Azure SQL instances.`, '\n\n'));
```